spring-kafka/src/main/java/org/springframework/kafka/annotation/EnableKafka.java

```diff
@@ -249,6 +250,6 @@
 @Target(ElementType.TYPE)
 @Retention(RetentionPolicy.RUNTIME)
 @Documented
-@Import(KafkaBootstrapConfiguration.class)
+@Import({KafkaBootstrapConfiguration.class, KafkaBootstrapConfiguration.KStreamsRegistrar.class})
```

Comment:
I thought about that in the beginning and that was our with you mind storm decision on the meeting last week.

But in the end I decided that  `@EnableKafka` is general annotation which says that the Spring Kafka infrastructure is switched on - feel free to to spill the code!

This our `enable KStream` case doesn't provide a lot of gain and it is conditional anyway.

I think it might be better to reconsider `KafkaListenerAnnotationBeanPostProcessor` registration if there is no `kafkaListenerContainerFactory`, but still have that very `@EnableKafka` and that's all.